sdk/core/azure-core/azure/core/pipeline/transport/_requests_basic.py

```diff
@@ -58,11 +59,11 @@ def _read_raw_stream(response, chunk_size=1):
             for chunk in response.raw.stream(chunk_size, decode_content=False):
                 yield chunk
         except ProtocolError as e:
-            raise requests.exceptions.ChunkedEncodingError(e)
-        except DecodeError as e:
-            raise requests.exceptions.ContentDecodingError(e)
+            raise ServiceResponseError(e, error=e)
+        except CoreDecodeError as e:
+            raise DecodeError(e, error=e)
```

Comment:
This scenario is being correctly handled in the other transports?

Reply:
Currently different transport raises different type of error. (That's the reason we need this change).

We want to uniform the errors so user can catch all by using except AzureError.